### PR TITLE
GAWB 1351

### PR DESCRIPTION
**Pull request by skwalker**
UI PR: https://github.com/broadinstitute/firecloud-ui/pull/625

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Make sure Swagger is updated if API changes
- [ ] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/main/resources/library/attribute-definitions.json [hunk 1]
```diff
@@ -231,8 +231,13 @@
     "library:lmsvn": {
       "title": "Library Metadata Schema Version Number",
       "description": "Library Metadata Schema Version Number",
-      "type": "integer",
-      "default": 3,
+      "current": 3,
```
Comment (skwalker) [inline src/main/resources/library/attribute-definitions.json]: I don't want to remove type from being an integer here but for some reason I was getting a number error on the UI when I tried to submit... tried using js/parseInt but couldn't get it to work.

**Comment by davidangb** [comment]
this is failing unit tests - you definitely need the "type" in there, otherwise it's not valid json-schema.

File: src/main/resources/library/attribute-definitions.json [hunk 2]
```diff
@@ -231,9 +231,16 @@
...
-      "default": 3,
-      "indexable": false
+      "current": 3,
+      "indexable": false,
+      "type": "integer"
+    },
+    "library:dulvn": {
+      "title": "Structured Data Use Limitations  Version Number",
+      "description": "Structured Data Use Limitations Version Number",
+      "current": 1,
```
Comment (davidangb) [inline src/main/resources/library/attribute-definitions.json]: why "current" instead of "default"? "default" is the json-schema standard.

Comment (rushtong) [reply src/main/resources/library/attribute-definitions.json]: 👍  to this comment

**Comment by rushtong** [comment]
👍  considering David's comment.

File: src/main/resources/library/attribute-definitions.json (hunk 2 above)
Comment (skwalker) [reply src/main/resources/library/attribute-definitions.json]: I was thinking that a default value is one you change only if it hasn't been set before, whereas we change to the current version number every time you catalog.  But I'm fine with calling it default.

**Pull request merged**